src/content/en/updates/2019/06/layoutNG.md

```diff
@@ -0,0 +1,328 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: A new layout engine for Chromium
+
+{# wf_published_on: 2019-06-06 #}
+{# wf_updated_on: 2019-06-06 #}
+{# wf_featured_image: /web/updates/images/2019/06/layoutNG-header.jpg #}
+{# wf_tags: new-in-chrome,layout,performance #}
```

Comment:
add `chrome76` 
remove `new-in-chrome` - we only use it for the New in Chrome video/update posts.

Reply:
let's add a new `layoutng` tag

Reply:
👍